text/rustconf-steering-committee.md

```diff
@@ -0,0 +1,50 @@
+- Feature Name: rustconf-steering-committee
+- Start Date: 12-27-2023
+- RFC PR: 
+- Rust Issue: N/A
+
+# Summary
+
+This RFC proposes creating a RustConf steering committee to represent the Rust Project’s perspectives and priorities throughout the RustConf 2024 planning process. 
+
+# Proposal
+
+The Rust Foundation staff team would like to propose that a RustConf steering committee composed of ~5 Rust Project members be formed with an agreed-upon process of member appointment and scope established.
+
+A Rust Project steering committee for RustConf could be defined as:
+
+A group of ~5 volunteers from within the Leadership Council and the Project Directors responsible for providing input and direction to the Rust Foundation staff team regarding RustConf 2024. The Rust Foundation will lean on the steering committee to advocate for areas of desired involvement besides the traditional UnConference Day and Program Committee. 
+
+Once a steering committee has been formed, we can establish a regular process of communication between the Rust Foundation, the steering committee, and other groups from within the Project as identified by the steering committee. 
+
+
+# Motivation
+
+While the Rust Foundation has served as a Diamond-level sponsor of RustConf since our inception in 2021, we have sought approval to use our staff and financial resources towards planning and managing RustConf in a larger way in 2024. We have gotten approval from our board and received [encouragement](https://rust-lang.zulipchat.com/#narrow/stream/335408-foundation/topic/RustConf.202024.20next.20steps/near/403266679) from the Project and Leadership Council to proceed in Q4.
+
+Although we have always intended for the Program Committee to remain independent (with management and facilitation provided by the Foundation), Ryan Levick commented that additional Project input throughout the planning process is desired. The staff team discussed this internally and decided that a steering committee would be the right vehicle to identify and drive the type of Rust Project input requested. 
+
+When the Rust Foundation staff team raised this idea to our Project Directors during our biweekly meeting, we were asked to outline our idea in an RFC.   
+
+
+# Practicalities
+
+Once this proposal is accepted, the Rust Foundation team can work with the Leadership Council and the Project Directors to carry out a volunteering process to form the steering committee. 
+
+Once the steering committee is staffed, we can agree upon a channel and cadence of communication. From there, we will be asking the steering committee to share a clear set of expectations about their preferred involvement in RustConf planning (if there are requests outside of Program Committee and UnConf involvement). We can discuss these desires and expectations in Zulip.
```

Comment:
> a clear set of expectations about their preferred involvement in RustConf planning

it might be good to give a rough list of what areas of planning exist when organizing such a conference. that would make it a lot easier for the volunteer team to figure out in which areas they would like to be involved. the list would not need to be part of the RFC, but it would probably be good to have during the volunteering process, so that people know what they would sign up for. :)